Src/Microsoft.Dynamic/Interpreter/InterpretedFrame.cs

```diff
@@ -119,9 +115,7 @@ public sealed class InterpretedFrame {
 
         #region Stack Trace
 
-        public InterpretedFrame Parent {
-            get { return _parent; }
-        }
+        public InterpretedFrame Parent => _parent;
```

Comment:
If no one is consuming `_parent` it would be nice to switch it to an auto property.